.github/workflows/ci.yml

```
@@ -12,11 +12,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scalaversion: ["2.11.12", "2.12.10", "2.13.1"]
+        scalaversion: ["2.11.12", "2.12.10", "2.13.1", "2.13.5", "3.0.0-RC1"]
```

Don't include 2.13.5. It won't work with 0.6.x, and we only need one 2.13.x version. 2.13.1 is there and works.
```suggestion
        scalaversion: ["2.11.12", "2.12.10", "2.13.1", "3.0.0-RC1"]
```